packages/dashboard/src/components/MultiCount/MultiCountStyles.ts

```diff
@@ -71,12 +72,21 @@ export const getStyles = (props: IMultiCountStyleProps): IMultiCountStyles => {
     },
     hoverCardAnnotationText: {
       marginLeft: '16px',
-      marginRight: '16px',
       fontFamily: 'Segoe UI'
     },
     hoverCardBodyText: {
       marginRight: '16px',
       fontFamily: 'Segoe UI'
+    },
+    customMessage: {
+      fontSize: '10px',
+      lineHeight: '12px',
+      fontFamily: 'Segoe UI',
+      fontWeight: 600,
+      marginTop: '13px',
+      marginLeft: '16px',
+      marginBottom: '8px',
+      opacity: '0.6'
```

Comment:
>opacity [](start = 6, length = 7)

0.1 right?

Reply:
This is for a different thing, this is for message in hover card. Although updated in right place